hideandseek/hider.cfg

```diff
@@ -1 +1,4 @@
-ghost_set_color 00ff04
\ No newline at end of file
+ghost_set_color 00ff04
+ghost_draw_through_walls 2
+sar_function found conds "?seeker=0" "ghost_set_color FFFFFF; noclip; ghost_message was found!" "say You must be a hider to mark yourself as found!"
+bind y found
```

Comment:
I don't like the idea of a key randomly getting rebinded, maybe just have the user bind it themself.